src/index.js

```diff
@@ -4,6 +4,9 @@ import PropTypes from 'prop-types';
 import isEqual from 'lodash.isequal';
 import { Picker } from '@react-native-picker/picker';
 import { defaultStyles } from './styles';
+import { Dimensions } from 'react-native';
+
+const IOS_MODAL_HEIGHT = 262;
```

Comment:
How did you come up with a value of 262?

Reply:
I tried to programatically measure the modal instead, but didn't succeed.

Instead i thoroughly tested this value on multiple iOS devices, since the height of the picker modal is the same on all (iOS) devices. Works the same way on iPhone 8:


https://user-images.githubusercontent.com/20173411/207729301-653bb935-0787-4969-a030-76a8fe1f55d5.mp4

Reply:
OK, thanks! Can you please add that context as a code comment?